internal/controller/orchestrator_controller.go

```diff
@@ -484,7 +382,7 @@ func (r *OrchestratorReconciler) UpdateStatus(ctx context.Context, orchestrator
 	logger := log.FromContext(ctx)
 	orchestrator.Status.Phase = phase
 	meta.SetStatusCondition(&orchestrator.Status.Conditions, condition)
-	//orchestrator.Status.Conditions = []metav1.Condition{condition}
+	//orchestrator.Status.Conditions = append(orchestrator.Status.Conditions, condition)
```

Comment:
why this comment was kept?

Reply:
Will remove comment. Initially I was testing different ways to add status condition. Don't need it anymore.

Reply:
Removed comment.